sql/hive/src/test/scala/org/apache/spark/sql/hive/execution/HiveCommandSuite.scala

```diff
@@ -58,7 +58,11 @@ class HiveCommandSuite extends QueryTest with SQLTestUtils with TestHiveSingleto
         |TBLPROPERTIES('prop1Key'="prop1Val", '`prop2Key`'="prop2Val")
       """.stripMargin)
     sql("CREATE TABLE parquet_tab3(col1 int, `col 2` int)")
-    sql("CREATE TABLE parquet_tab4 (price int, qty int) partitioned by (year int, month int)")
+    sql(
+      """
+        |CREATE TABLE parquet_tab4 (price int, qty int) partitioned by (year int, month int)
+        |STORED AS PARQUET
```

Comment:
is it a necessary change?

Reply:
It's modified because I randomly chose it to make sure the insert into partitioned table can be safely converted.
However it should already covered in other test cases.

So it's neutral change. I can revert it if you think it's unnecessary.

Reply:
reverted